db/database.go

```diff
@@ -61,11 +61,11 @@ var (
 	DefaultDeltaSyncRevMaxAge = uint32(60 * 60 * 24) // 24 hours in seconds
 )
 
-var DefaultCompactInterval = uint32(1440) // Default compact interval in minutes
+var DefaultCompactInterval = uint32(60 * 60 * 24) // Default compact interval in seconds = 1 Day
 
 const (
-	CompactIntervalMin = 60
-	CompactIntervalMax = 86400
+	CompactIntervalMinDays = float32(0.042) //1 Hour in days
+	CompactIntervalMaxDays = float32(60)    //60 Days in seconds
```

Comment:
"in seconds" should be removed now

```suggestion
	CompactIntervalMaxDays = float32(60)
```